core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-palette/dot-palette.component.scss

```diff
@@ -2,10 +2,10 @@
     background-color: white;
     display: block;
     height: 100%;
+    overflow: hidden;
```

Comment:
if we hide the overflow how can we scroll if the component content is more than container size?